web/src/main/java/org/cbioportal/genome_nexus/web/AnnotationController.java

```diff
@@ -58,12 +58,18 @@
 @Validated
 public class AnnotationController
 {
-    private final VariantAnnotationService variantAnnotationService;
+    private final VariantAnnotationService hgvsAnnotationService;
+    private final VariantAnnotationService dbsnpAnnotationService;
+    private final GenomicLocationAnnotationService genomicLocationAnnotationService;
 
     @Autowired
-    public AnnotationController(VariantAnnotationService variantAnnotationService)
+    public AnnotationController(@Qualifier("VariantAnnotationServiceImplHgvs") VariantAnnotationService hgvsAnnotationService,
```

Comment:
If we name the implementation class as `HgvsVariantAnnotationService` and the variable as `hgvsVariantAnnotationService`, we may get away without using `@Qualifier("VariantAnnotationServiceImplHgvs")`, and `@Service(value="VariantAnnotationServiceImplHgvs")` at all. Not hundred percent sure if it works though :)